### PR TITLE
change class of end game player name

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Change the player username class to be the same as in log.

New: image

Previous:
image

### CODE REVIEW

**Comment by nwai90** [comment]
This looks great! Makes the player names a lot more visible 👍

**Comment by kberg** [comment]
Looks great.

File: src/components/GameEnd.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export const GameEnd = Vue.component("game-end", {
         isSoloGame: function (): boolean {
             return this.player.players.length === 1;
         },
-        getPlayerColorStyle: function (player: PlayerModel): string {
-            return "color: " + player.color;
+        getEndGamePlayerColorClass: function (player: PlayerModel): string {
```
Comment (chosta) [inline src/components/GameEnd.ts:21]: @vsrisuknimit this could be :) 
`playerColorClass("bg", player.color)`

p.s. maybe add it as a todo for a next PR and do a separate commit for it
